modules/storages/spec/requests/api/v3/storages/storages_spec.rb

```diff
@@ -98,5 +98,16 @@
     context 'when storages module is inactive', with_flag: { storages_module_active: false } do
       it_behaves_like 'not found'
     end
+
+    context 'when Storage connected to a OAuth2 Authorization Server' do
+      subject { last_response.body }
+
+      let(:current_user) { create(:admin) }
+
+      it 'returns an error authorization state' do
+        expect(subject).to be_json_eql("urn:openproject-org:api:v3:storages:authorization:Error".to_json)
+                         .at_path('_links/authorizationState/href')
```

Comment:
This only tests the error case. I'd advise to also test the happy case which requires stubbing that check request to Nextcloud